### PR TITLE
manual and stabilised flight modes are the same

### PR DESCRIPTION
**Pull request by barzanisar**

### CODE REVIEW

**Comment by SinigerVTL** [comment]
Fact! But that was the most controversial issue I had with the consistency of modes naming...

File: en/concept/flight_modes.md [hunk 1]
```diff
@@ -19,10 +19,9 @@ The sections below provide an overview of the modes, followed by a [flight mode
...
-  * **STABILIZED** The pilot's inputs are passed as roll and pitch *angle* commands and a yaw *rate* command. Throttle is passed directly to the output mixer. The autopilot controls the attitude, meaning it regulates the roll and pitch angles to zero when the RC sticks are centered, consequently leveling-out the attitude. However, in this mode the position of the vehicle is not controlled by the autopilot, hence the position can drift due to wind.
+  * **MANUAL/STABILIZED** The pilot's inputs are passed as roll and pitch *angle* commands and a yaw *rate* command. Throttle is passed directly to the output mixer. The autopilot controls the attitude, meaning it regulates the roll and pitch angles to zero when the RC sticks are centered, consequently leveling-out the attitude. However, in this mode the position of the vehicle is not controlled by the autopilot, hence the position can drift due to wind.
```
Comment (hamishwillee) [inline en/concept/flight_modes.md]: 1. I'd move this point up the top - as it is the most "basic" case (ie not at the bottom). 
2. I'd explicitly add a note that they are the same mode for MC at the bottom of this section.
3. Can you please create a matching change in the PX4 user guide?

File: en/concept/flight_modes.md [hunk 2]
```diff
@@ -19,9 +19,12 @@ The sections below provide an overview of the modes, followed by a [flight mode
...
   * **RATTITUDE** The pilot's inputs are passed as roll, pitch, and yaw *rate* commands to the autopilot if they are greater than the mode's threshold, i.e. if the RC sticks are a certain distance away from the center position.  If not the inputs are passed as roll and pitch  *angle* commands and a yaw *rate* command.  Throttle is passed directly to the output mixer. In short, the autopilot acts as an angular rate controller when the RC sticks are away from center (like in the ACRO mode), whereas when the RC sticks are centered, the autopilot acts as an attitude controller (like in the Stabilized mode).
-  * **MANUAL/STABILIZED** The pilot's inputs are passed as roll and pitch *angle* commands and a yaw *rate* command. Throttle is passed directly to the output mixer. The autopilot controls the attitude, meaning it regulates the roll and pitch angles to zero when the RC sticks are centered, consequently leveling-out the attitude. However, in this mode the position of the vehicle is not controlled by the autopilot, hence the position can drift due to wind.
+  
+  > **Note** For Multirotors, Manual and Stabilized modes are the same.
```
Comment (hamishwillee) [inline en/concept/flight_modes.md]: Looks great, but IMO this should move up to be under the **MANUAL/STABILIZED** section. Thoughts?

**Comment by hamishwillee** [comment]
Thank you. Merging now :-)

**Pull request merged**